contexts/credentials/v2

```diff
@@ -1,6 +1,7 @@
 {
   "@context": {
     "@protected": true,
+    "@vocab": "https://www.w3.org/2018/credentials#private-claim-",
```

Comment:
It's sad to see `2018` stuck in here forever.

I've opted to provide vocab expansions with a fragment prefix, so that no changes to W3C directories is needed.

This will lead to term definitions that look like this:

```
https://www.w3.org/2018/credentials#private-claim-failedToDefineThisTermInASecondContext
```

For cases like this:

 {
  "@context": "https://www.w3.org/2018/credentials/v2",
  "id": "http://example.edu/credentials/1872",
  "type": ["VerifiableCredential"],
  "issuer": "did:example:123",
  "issuanceDate": "2010-01-01T19:23:24Z",
  "credentialSubject": {
    "id": "did:example:456",
    "failedToDefineThisTermInASecondContext": "🧠💎"
  }
}

Reply:
> It's sad to see `2018` stuck in here forever.

I agree that `2018` is not a good value there and wished we could have gone another direction in the 1.0 WG. I  also believe there's been discussion around whether we may be able to change that and replace it with `ns`. So it's conceivable that we wouldn't be stuck with it forever, but if we're going to make that change, it should definitely be before 2.0 goes to REC.

Reply:
I filed https://github.com/w3c/vc-data-model/issues/1002#issue-1497487224 to track that discussion, I would love to fix this before 2.0 goes to REC.